plugins/woocommerce-admin/client/tasks/tasks.tsx

```diff
@@ -15,29 +15,20 @@ import { recordEvent } from '@woocommerce/tracks';
  */
 import { DisplayOption } from '~/activity-panel/display-options';
 import { Task } from './task';
-import { TasksPlaceholder } from './placeholder';
+import { TasksPlaceholder, TasksPlaceholderProps } from './placeholder';
 import './tasks.scss';
 import { TaskListProps } from './task-list';
+import { TaskList } from './task-list';
+import { TaskList as TwoColumnTaskList } from '../two-column-tasks/task-list';
+import TwoColumnTaskListPlaceholder from '../two-column-tasks/placeholder';
 import '../two-column-tasks/style.scss';
+import { getAdminSetting } from '~/utils/admin-settings';
 
 export type TasksProps = {
 	query: { task?: string };
 };
 
-const TaskList = lazy(
-	() => import( /* webpackChunkName: "task-list" */ './task-list' )
-);
-
-const TwoColumnTaskList = lazy(
-	() =>
-		import(
-			/* webpackChunkName: "two-column-task-list" */ '../two-column-tasks/task-list'
-		)
-);
```

Comment:
Using chunks for the task list was actually causing a delay, as retrieving the chunks got stuck behind some other requests that took longer.